tachyon/crypto/commitments/mixed_matrix_commitment_scheme.h

```diff
@@ -38,6 +43,46 @@ class MixedMatrixCommitmentScheme {
     Derived* derived = static_cast<Derived*>(this);
     return derived->DoCommit(std::move(matrices), result);
   }
+
+  [[nodiscard]] bool CreateOpeningProof(
+      size_t index, std::vector<std::vector<Field>>* openings,
+      Proof* proof) const {
+    const Derived* derived = static_cast<const Derived*>(this);
+    return derived->DoCreateOpeningProof(index, openings, proof);
+  }
+
+  const std::vector<math::RowMajorMatrix<Field>>& GetMatrices() const {
+    const Derived* derived = static_cast<const Derived*>(this);
+    return derived->DoGetMatrices();
+  }
+
+  std::vector<size_t> GetRows() const {
+    return base::Map(GetMatrices(),
+                     [](const math::RowMajorMatrix<Field>& matrix) {
+                       return matrix.rows();
+                     });
+  }
+
+  size_t GetMaxRow() const {
```

Comment:
```suggestion
  size_t GetMaxRows() const {
```

Reply:
If you see above `GetRows()`, it returns a vector and if I modify it to `GetMaxRow()`, then it looks like it should return a vector as well.